supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -173,6 +195,12 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
+  private def generateGcsEgressCheckScript(inputs: List[PipelinesApiInput])
```

Comment:
I think this may need to exclude inputs being provided by reference disks as `generateGcsLocalizationScript` does. e.g. we have reference disks for several inputs from `gs://broad-references`. This bucket is US multi-regional, but `continental` regionality would exclude researchers in Australia from being able to use these inputs if the localization check doesn't realize they're being provided by the reference disk and do not require localization.